printer.py

```diff
@@ -5,4 +5,4 @@
 
 with open(file_name, 'r') as file:
 	for line in file:
-		print(line)
+		print(len(line), line)
```

Comment:
please update this string to be more readable, like:
f'lenght={len(line)}, string={line}'